src/main/java/edu/wisc/my/messages/model/Message.java

```diff
@@ -291,6 +315,55 @@ public ActionButton getConfirmButton() {
   public void setConfirmButton(ActionButton confirmButton) {
     this.confirmButton = confirmButton;
   }
+  
+  public boolean isValidToday(){
+
+   Date today = new Date();
+   
+   // Placeholder date format string, 
+   // can be overridden with an application property.
+   String formatString = "yyyy-mm-dd";
+   
+   if(env == null){
+     logger.error("NO ENV VARIABLE");
+   } else {
+      formatString = env.getProperty("default.date.format");
```

Comment:
YAGNI? `yyyy-mm-dd` is the ISO format, let's just hard code to that format until a requirement comes along to do something different?